src/pack/pack.js

```diff
@@ -0,0 +1,19 @@
+import path from 'path';
+import fs from 'fs-extra';
+import childProcessP from 'child-process-promise';
+
+const exec = childProcessP.exec
+
+export default async function pack(cwd: string, outputPath: string, force: boolean = false): string {
```

Comment:
the force logic is not forcing but overriding, change the name.
And throw error if the file exists and override is false